canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -537,7 +531,6 @@ VirtualizedTable.propTypes = {
 		PropTypes.func.isRequired,
 		PropTypes.number.isRequired
 	]),
-	onRowDoubleClick: PropTypes.func,
```

Comment:
I think we should keep this.  It's optional so nothing should happen if we don't pass in `onRowDoubleClick` from the exression builder.